api/cpp/include/slint.h

```diff
@@ -124,6 +124,11 @@ upgrade_item_weak(const cbindgen_private::ItemWeak &item_weak)
     }
 }
 
+inline void slint_debug(const SharedString &str)
```

Comment:
Just curious why this is called `slint_debug` instead of just `debug` like the rust equivalent function.

Reply:
Good point, it should just be called `debug`. I forgot to remove the `slint_` prefix when I moved from a direct call into the ffi to this wrapper (because of pointer vs. reference). I'll change that.